pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
```

Comment:
what are all of these changes to the lockfile? is it just that pnpm changed how some stuff works in a recent version?

Reply:
This happened when I installed the stackblitz sdk using pnpm

Reply:
Can you make sure you're using pnpm@8.5.1 (as specified in package.json, then run

```
git checkout next pnpm-lock.yaml
pnpm i
```

That should remove any unnecessary diffs